packages/core/src/dev/activation.ts

```diff
@@ -160,6 +160,11 @@ export async function activate(ctx: vscode.ExtensionContext): Promise<void> {
             webviewOptions: {
                 retainContextWhenHidden: true,
             },
+        }),
+        vscode.commands.registerCommand('aws.explorer.setLoginService', (serviceToShow?: string) => {
```

Comment:
this file is for dev tools, it isn't activated if the user does not have `aws.dev.*` settings